apps/foundation-demo/src/app/misc/header/components/header/header.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="header">
+  <span class="header__open-icon" (click)="toggleShowSidebar()"><i class="fa fa-bars"></i></span>
+  <span class="header__logo-icon" (click)="navigateToHomepage()"><img src="../../../../../assets/logo-icon.jpg"></span>
+  <span class="header__cart-icon"><i class="fa fa-shopping-cart"></i></span>
+  <sidebar [showSidebar]="showSidebar$ | async"></sidebar>
```

Comment:
I'm not exactly sure why `sidebar` is inside header. What's the reasoning?

Reply:
I could see why you don't like it. It seemed like the only place to put it at the time, but now that I've been looking at how graycore works, I see that it would be better to have a `default-template` component that goes like this:

```
<header></header>
<sidebar></sidebar>

<router-outlet></router-outlet>
```

Reply:
moved sidebar out of header